src/components/Responsive/Hidden/Hidden.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, ReactElement } from 'react';
+import intersection from 'lodash/intersection';
+
+import Context from '../Context';
+
+interface HiddenProps {
+  breakpoints: string[];
```

Comment:
Breakpoints wasn't previously required. the lodash `intersection` method is null / undefined safe, and just returns an empty array.

Small, but I think this merits a fix.